Sources/ExyteChat/ChatView/Attachments/AttachmentsEditor.swift

```diff
@@ -10,42 +10,42 @@ import ExyteMediaPicker
 import ActivityIndicatorView
 
 struct AttachmentsEditor<InputViewContent: View>: View {
-
+    
```

Comment:
Please don't add these, this can be changed in xcode settings

Reply:
will fix

Reply:
It would be good to see about getting https://github.com/exyte/Chat/pull/126 merged, that way I could just run the command to meet the projects formatting